modules/nf-core/presto/filterseq/main.nf

```diff
@@ -0,0 +1,47 @@
+process PRESTO_FILTERSEQ {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::presto=0.7.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/presto:0.7.1--pyhdfd78af_0':
+        'biocontainers/presto:0.7.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*_quality-pass.fastq"),  emit: reads
+    path "*_command_log.txt" , emit: logs
+    path "versions.yml" , emit: versions
+    path "*.tab" , emit: log_tab
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    FilterSeq.py quality -s $reads --outname ${meta.id} --log ${reads.baseName}.log --nproc ${task.cpus} $args > ${meta.id}_command_log.txt
```

Comment:
Is it possible to arrange a python script to have line breaks in its parameters? A lot of modules do it with bash commands to increase readability.